odml/section.py

```diff
@@ -153,6 +163,23 @@ def parent(self):
         """the parent section, the parent document or None"""
         return self._parent
 
+    @parent.setter
+    def parent(self, new_parent):
+        if new_parent is None:
+            return
+        elif self._validate_parent(new_parent):
+            self._parent = new_parent
+            self._parent.append(self)
+        else:
+            raise ValueError("odml.Section.parent: passed value is not of consistent type! \n"
+                             "odml.Document or odml.Section expected")
+
+    def _validate_parent(self, new_parent):
+        from odml.doc import BaseDocument
```

Comment:
Same comment as above. Do we have issues with import cycles if this is moved to the top?